modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,94 @@
+
+## Scope
+
+The Candidate Parameters module stores all information 
+for a candidate that are not timepoint dependent. All data 
+being collected during a visit should be entered as an instrument 
+at the visit level.
+
+## Permissions
+
+The Candidate Parameters module uses the following permissions. Any one of them
+is sufficient to have access to the module.
+
+ - candidate_parameter_view
+    - This permission gives the user viewing access to the module and all its tabs.
+
+ - candidate_parameter_edit
+    - This permission gives the user editing access to the module and all its tabs.
+
+## Configurations
+
+The candidate parameters module has the following configurations that affect its usage
+
+ - useProband 
+    - This setting displays/hides the Proband tab of the module.
+
+ - useFamilyID
+    - This setting displays/hides the Family tab of the module.
+
+ - useConsent 
+    - This setting displays/hides the Consent tab of the module.
+ 
+ #### Additional Parameters
+ In addition to the Settings above, it is possible to add candidate parameters to the Candidate Info page. To do so, values must be added in SQL following the example below. These queries will add a single additonal dropdown with label _"Visit plan for candidate"_, field name _"candidate_plan"_, and options _"6month"_, _"12month"_. When candidate information is entered for this field, the value is saved in the `parameter_candidate` table in SQL.
+ 
+ Example:
+ ```
+ INSERT INTO `parameter_type_category` (Name, Type) VALUES ('Candidate Parameters','Metavars');
+ INSERT INTO parameter_type (Name, Type, Description, SourceFrom, Queryable) VALUES 
+  ('candidate_plan', "enum('6month', '12month')", 'Visit plan for candidate', 'parameter_candidate', 1);
+ 
+  INSERT INTO parameter_type_category_rel (ParameterTypeID, ParameterTypeCategoryID) SELECT 
+  pt.ParameterTypeID, ptc.ParameterTypeCategoryID FROM parameter_type_category ptc, 
+  parameter_type pt WHERE ptc.Name='Candidate Parameters' AND pt.Name='candidate_plan';
+ ```
+ 
+ #### Additional Consents
+ It is possible to have multiple consent types in the Consent Status tab. In order to add consents, first make sure the useConsent option is set to `true`, second fillow the instructions below.
+ 
+ 1. Populate the config file with additional Consent tagsets. For example, for consent to draw blood:
+
+         ```xml
+         <ConsentModule>
+             <useConsent>true</useConsent>
+             <Consent>
+                 <name>study_consent</name>
+                 <label>Consent to Study</label>
+             </Consent>
+             <Consent>
+                 <name>draw_blood_consent</name>
+                 <label>Consent to draw blood</label>
+             </Consent>
+         </ConsentModule>
+         ```
+ 2. Then, alter both participant_status and consent_info_history table schemas, adding 3 columns for the new consent type, to store consent status, date and withdrawal status. For example:
+
+         ```sql
```

Comment:
I don't know what is the convention for code examples but in the example for "additional parameters" there is not sql specified after ```. Would be good to stay consistent across the document.

Also, for some reason, if using ```sql, then it looks like that in the .md file.
![screen shot 2018-04-09 at 11 41 46 am](https://user-images.githubusercontent.com/1402456/38507617-05a4a306-3beb-11e8-9492-8f020f5868e4.png).
Is there a way to not have the backtilt show?

Reply:
done, markdown includes the backticks when you over-indent a paragraph